azure-pipelines.yml

```diff
@@ -238,11 +238,14 @@ stages:
 
       - template: ci-templates/native-build-steps.yaml
         parameters:
-          timeoutInMinutes: 20
+          timeoutInMinutes: 25
           modules:
             - elytron-security-oauth2
             - elytron-security
             - keycloak
+            - vault
+            - vault-agroal
+            - vault-app
```

Comment:
You only need to add there the modules with native tests.

Reply:
did not know. thanks.
done